programs/sll_remove_dups.py

```diff
@@ -0,0 +1,48 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        self.head = None
```

Comment:
It is clever to have nodes point to the head. However, it becomes a problem to have all the nodes up to date as to what the head is. Every time that the head changes, you would have to update every single node so that it is up to date.

No need to point to head here.